spec/spec_helper.rb

```diff
@@ -3,3 +3,17 @@
 root_dir = File.expand_path('..', __dir__)
 IMAGES_DIR = File.join(root_dir, 'doc/ex/images')
 SUPPORT_DIR = File.join(root_dir, 'spec', 'support')
+
+module Magick
+  module_function
+
+  def magick_lib_version
+    version = Magick::Magick_version.split[1]
+    version.slice!(/-\d+$/)
```

Comment:
Being that ruby is moving [in the direction of immutable strings](https://bugs.ruby-lang.org/issues/11473), I wonder if we want to do this in a non-destructive fashion. Here's one idea:

```rb
version.split('-').first
```

I don't know how reliable the `Magick_version` string is, but another idea is that we could condense it down to one line like:

```rb
Magick::Magick_version.match(/\d+\.\d+\.\d+/)[0]
```

or:

```rb
Magick::Magick_version.scan(/\d+\.\d+\.\d+/).first
```